examples/research_projects/pytorch_xla/README.md

```diff
@@ -5,15 +5,7 @@ The `train_text_to_image_xla.py` script shows how to fine-tune stable diffusion
 It has been tested on v4 and v5p TPU versions. Training code has been tested on multi-host. 
 
 This script implements Distributed Data Parallel using GSPMD feature in XLA compiler
-where we shard the input batches over the TPU devices. 
-
-As of 9-11-2024, these are some expected step times.
-
-| accelerator | global batch size | step time (seconds) |
-| ----------- | ----------------- | --------- |
-| v5p-128 | 1024 | 0.245 |
-| v5p-256 | 2048 | 0.234 |
-| v5p-512 | 4096 | 0.2498 |
```

Comment:
Can we update the corrected numbers instead of deleting the table?

Reply:
It will take some time to collect these. Can we push these changes for now with an action item to add this table in the future?

Reply:
I won't prefer that. Since we already had these values and communicated about it, I would prefer if the PR indicated the changes. Cc: @tengomucho as well.